doc/sphinx-guides/source/installation/shibboleth.rst

```diff
@@ -39,28 +39,10 @@ Install Shibboleth
 
 Installing Shibboleth will give us both the ``shibd`` service and the ``mod_shib`` Apache module.
 
-Enable Shibboleth Yum Repo
-^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-This yum repo is recommended at https://wiki.shibboleth.net/confluence/display/SHIB2/NativeSPLinuxRPMInstall
-
-``cd /etc/yum.repos.d``
-
-Install ``wget`` if you don't have it already:
-
-``yum install wget``
-
-If you are running el8 (RHEL/derivative 8):
-
-``wget http://download.opensuse.org/repositories/security:/shibboleth/CentOS_8/security:shibboleth.repo``
```

Comment:
Should we do anything with `conf/vagrant/etc/yum.repos.d/shibboleth.repo`?

It's referenced in `scripts/vagrant/setup.sh`.

Reply:
Done in 42a39d1. Thanks!